apk-generator/v2/config.py

```diff
@@ -22,6 +22,7 @@ class Config:
     CELERY_BROKER_URL = os.environ.get('REDIS_URL', 'redis://localhost:6379/0')
     CELERY_RESULT_BACKEND = CELERY_BROKER_URL
     SECRET_KEY = os.environ.get('SECRET_KEY', 'JQgeEYk9b4VjxhJP')
+    PORT = os.environ.get('PORT', '8080')
```

Comment:
Like i had said ... Also check for an env variable `ANDROID_GENERATOR_SERVICE_PORT`. 

Priority should be 
1. `PORT` env variable (user override)
2. `ANDROID_GENERATOR_SERVICE_PORT` env variable (auto-set for kubernetes deployments)
3. `8080` (default)